src/main/resources/view/SaleCard.fxml

```diff
@@ -38,7 +38,7 @@
                                     <graphic>
                                         <ImageView fitHeight="17.0" fitWidth="17.0" pickOnBounds="true" preserveRatio="true">
                                             <image>
-                                                <Image url="@../images/alarm-clock.png" />
+                                                <Image url="@../images/calendar.png" />
                                             </image>
                                         </ImageView>
                                     </graphic>
```

Comment:
Can you design your OWN UI as we agreed earlier. Simply copy-pasting without significant changes undermines both of our marks. At this stage of product development, you should have your own Ui designed.